tests/fixture_overwrites.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+import pytest
+from raiden_contracts.contract_manager import ContractManager, CONTRACTS_SOURCE_DIRS
+from raiden_libs.blockchain import BlockchainListener
+
+
+@pytest.fixture(scope='session')
```

Comment:
we can even have some tests disabled conditionally, depending on availability of libraries. See
https://github.com/raiden-network/microraiden/blob/master/microraiden/test/conftest.py#L14

Reply:
Did you have anything special in mind?

Reply:
I mean the tests that require compilation - if `solc` or `raiden-contracts` is unavailable, they can be skipped. Not sure if that's required, probably not.

Reply:
Lets add this later or if it goes out hand